src/preload.py

```diff
@@ -814,18 +842,30 @@ def get_docker_service_file_path(folder):
             return fpath
 
 
-def get_docker_service_file_content(image=None):
+def get_rootA_file_contents(image=None):
+    file_contents = {
+        "docker_service": "",
+        "etc_issue": "",
+    }
```

Comment:
This is an optimization to avoid mounting the rootA partition twice (once for reading `/lib/systemd/system/balena.service` and then again for reading `/etc/issue`). The partition is mounted only once, and contents of those files saved to the `file_contents` object which is returned.